github-activity-tracker/backend/migrations/010_add_pr_files_changed_columns.sql

```
@@ -0,0 +1,8 @@
+-- Track file changes per PR event and aggregate per repo/user.
+ALTER TABLE activity_events ADD COLUMN IF NOT EXISTS files_changed INTEGER;
+
+ALTER TABLE repo_users ADD COLUMN IF NOT EXISTS pr_files_changed_total INTEGER DEFAULT 0;
+
+CREATE INDEX IF NOT EXISTS idx_activity_events_pr_files_changed_backfill
+  ON activity_events (repo_id)
+  WHERE event_type = 'pr' AND files_changed IS NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)